go.mod

```diff
@@ -109,3 +109,6 @@ replace (
 	cloud.google.com/go => cloud.google.com/go v0.81.0
 	github.com/opencontainers/image-spec => github.com/opencontainers/image-spec v1.0.2
 )
+
+// v0.3.0 was retagged: cd604c1b597558ea045a79c4f80a8c780909801b -> 85653575c7dafb6b06548478ee1dc61ac5905d00
+retract v0.3.0
```

Comment:
Should we put the retract line at Line 101 retract parentheses?

Reply:
Thanks, updated